### DISCUSSION TITLE
Need assistance with compiling

### DISCUSSION THREAD

**Discussion by The-Evil-Pickle** [KadeArchive/Kade-Engine#147] [Q&A] [answered]
Trying to compile source code, but having issues. I've downloaded everything as instructed, but when I try to build, I get this:

PS D:\Tobias\Downloads\Kade-Engine-1.4.1-KE\Kade-Engine-1.4.1-KE> lime test html5 -debug
D:/HaxeToolkit/haxe/lib/flixel/4,8,1/flixel/group/FlxGroup.hx:473: characters 80-91 : Type not found : ObjectClass
D:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
D:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
D:/HaxeToolkit/haxe/lib/flixel/4,8,1/flixel/group/FlxGroup.hx:852: characters 19-30 : Type not found : ObjectClass
D:/HaxeToolkit/haxe/lib/flixel/4,8,1/flixel/system/frontEnds/PluginFrontEnd.hx:47: characters 19-28 : Type not found : ClassType
D:/HaxeToolkit/haxe/lib/flixel/4,8,1/flixel/system/frontEnds/PluginFrontEnd.hx:94: characters 20-29 : Type not found : ClassType

Notes: 
- I have not  yet made any changes to the code, I'm just trying to compile the latest release from the source code.

Any advice would be greatly appreciated!

**Comment by isakube** [comment, marked as answer]
are you running haxe 4.1.5? downgrading fixed these exact deprecation warnings for me :)
